litmus-portal/frontend/src/containers/layouts/Scaffold/styles.ts

```diff
@@ -12,14 +12,14 @@ const useStyles = makeStyles((theme: Theme) => ({
     gridTemplateAreas: '"header header" "sidebar content"',
 
     '& ::-webkit-scrollbar': {
-      width: '0.4rem',
```

Comment:
We are already increasing the font size in buttons 14px->16px and the font style to Inter to make things more visible..would it be a good idea to reduce the size of the scrollbar...?

Reply:
The problem is doing this overriding all scrollbars graph legend tables etc, we need to override either in litmus-ui and all child components getting affected or inject this scrollbar css component/page wise where required @vanshBhatia-A4k9